docs/ref/pgcopydb_restore.rst

```diff
@@ -290,8 +290,8 @@ First, using ``pgcopydb restore schema``
    09:54:37 20401 INFO  Found a stale pidfile at "/tmp/target//pgcopydb.pid"
    09:54:37 20401 WARN  Removing the stale pid file "/tmp/target//pgcopydb.pid"
    09:54:37 20401 INFO  Using pg_restore for Postgres "12.9" at "/Applications/Postgres.app/Contents/Versions/12/bin/pg_restore"
-   09:54:37 20401 INFO   /Applications/Postgres.app/Contents/Versions/12/bin/pg_restore --dbname 'port=54314 dbname=demo' --clean --if-exists /tmp/target//schema/pre.dump
-   09:54:38 20401 INFO   /Applications/Postgres.app/Contents/Versions/12/bin/pg_restore --dbname 'port=54314 dbname=demo' --clean --if-exists --use-list /tmp/target//schema/post.list /tmp/target//schema/post.dump
+   09:54:37 20401 INFO   /Applications/Postgres.app/Contents/Versions/12/bin/pg_restore --dbname 'port=54314 dbname=demo' --clean --if-exists /tmp/target//schema/schema.dump
+   09:54:38 20401 INFO   /Applications/Postgres.app/Contents/Versions/12/bin/pg_restore --dbname 'port=54314 dbname=demo' --clean --if-exists --use-list /tmp/target//schema/post.list /tmp/target//schema/schema.dump
```

Comment:
I think this is missing the `--section pre-data` and `--section post-data` command line options. Feel free to reproduce these outputs from scratch on your system.

Reply:
That makes sense. Alternatively, we can remove these log messages from the docs completely. What do you think?

Reply:
As a user I like being able to make sense of what the software is doing by reading the logs, and it's even better if I didn't need to setup anything to be able to read through logs.

Reply:
Because it has a considerable maintenance cost, I am not a fan of it. But I will update it as it already looks out-of-date.

Reply:
We can work separately on maintenance cost. There is no substitute to having good docs. That's just a lot of work, and very useful, and we are the ones to make it happen.